photon-ml/src/integTest/scala/com/linkedin/photon/ml/estimators/GameEstimatorTest.scala

```diff
@@ -0,0 +1,305 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
```

Comment:
Actually, were don't really need to update these. Copyright lasts for a really long time.